raspberry-pi-workers/README.md

```diff
@@ -0,0 +1,15 @@
+# zse-raspberry-pi-workers
+dockerized buildbot-workers for various versions of the raspberry-pi
+
+## Prerequisites
+- you have to have docker and docker-compose installed.
+- add your unprivileged user to the docker group
+
+## Setup
+* clone / copy this repository to a raspberry-pi.
```

Comment:
A physical Raspberry Pi?

Reply:
Yes - added hint. Also, the instructions are now enumerated.